src/documents/drafts/62304-mapping.md

```diff
@@ -106,6 +106,8 @@ OpenRegulatory templates).
 
 ## Document Approval
 
-| Activity | Date       | Role | Name        | Initials |
-| -------- | ---------- | ---- | ----------- | -------- |
-| Creation | 2022-02-28 | CEO  | Simon Biggs | SB       |
+| Activity | Date       | Role                  | Name             | Initials |
```

Comment:
This is a record and doesn't need to be approved, so it only needs creation which you can take from the git history. So no signing table needed.

Reply:
Awesome, thanks @olieidel :slightly_smiling_face: